app-emulation/guestfs-tools/guestfs-tools-1.46.1-r1.ebuild

```diff
@@ -121,6 +121,9 @@ src_configure() {
 		append-ldflags "-Wl,-R${ESYSROOT}/usr/$(get_libdir)/xcrypt"
 	fi
 
+	# Disable a warning which aborts the build
+	sed -i 's|CDEFLMPSUVYZX+52-3|+C+D+E+F+L+M+P+S+U+V+Y+Z+X+52-3-6 -w -6|' configure
```

Comment:
|| die on external commands which may fail. apart from that, looks fine, please merge. ocaml stuff tends to need this sort of hack